doctor/cores/api_ingress.py

```
@@ -0,0 +1,355 @@
+
+logger = logging.getLogger(__name__)
+User = get_user_model()
+
+
+class DoctorAPIIngressCore:
+    """
+    هسته مدیریت ورودی API برای اپ Doctor
+    مسئول validation، authentication، rate limiting و logging اپ دکتر
+    """
+    
+    def __init__(self):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)